docker/hvd/build_all.sh

```diff
@@ -30,7 +30,14 @@ cd $curr_dir/hvd
 set -eu
 
 image_tag=""
-pth_version=${PTH_VERSION:-1.6.0-cuda10.1-cudnn7}
+
+if [[ -z "${PTH_VERSION}" ]]; then
+    echo "PTH_VERSION is not set"
+    exit 1
+else
+    pth_version=${PTH_VERSION}
+fi
+
 hvd_version=${HVD_VERSION:-v0.21.0}
```

Comment:
Let's remove default values for HVD_VERSION and MSDP_VERSION too

Reply:
yes indeed